packages/player/src/components/shortcut.tsx

```diff
@@ -92,186 +105,204 @@ export const Shortcut: React.FC<ShortcutProps> = ({
-            source: 'shortcut',
-          })
-        },
-      },
-      {
-        keyCode: 36, // Home
-        handle: () => {
-          if (!hasStarted) {
-            return
-          }
-          videoService.send({
-            type: 'SEEKING',
-            seekingTime: 0,
-            source: 'shortcut',
-          })
-        },
-      },
-      {
-        keyCode: 35, // End
-        handle: () => {
-          if (!hasStarted) {
-            return
-          }
-          // Go to end of video
-          videoService.send({
-            type: 'SEEKING',
-            seekingTime: duration,
-            source: 'shortcut',
-          })
-        },
-      },
-      {
-        keyCode: 38, // Up arrow
-        handle: () => {
-          // Increase volume 5%
-          let v = volume + 0.05
-          if (v > 1) {
-            v = 1
-          }
-          videoService.send({
-            type: 'VOLUME_CHANGE',
-            volume: v < 1.0 ? v : 1.0,
-            source: 'shortcut',
-          })
-        },
-      },
-      {
-        keyCode: 40, // Down arrow
-        handle: () => {
-          // Decrease volume 5%
-          let v = volume - 0.05
-          if (v < 0) {
-            v = 0
-          }
-          //TODO difference between volume down and volume off actions
-          videoService.send({
-            type: 'VOLUME_CHANGE',
-            volume: v > 0 ? v : 0,
-            source: 'shortcut',
-          })
-        },
-      },
-      {
-        keyCode: 190, // Shift + >
-        shift: true,
-        handle: () => {
-          // Increase speed
-          let newPlaybackRate = playbackRate
-          if (playbackRate >= 1.5) {
-            newPlaybackRate = 2
-          } else if (playbackRate >= 1.25) {
-            newPlaybackRate = 1.5
-          } else if (playbackRate >= 1.0) {
-            newPlaybackRate = 1.25
-          } else if (playbackRate >= 0.5) {
-            newPlaybackRate = 1.0
-          } else if (playbackRate >= 0.25) {
-            newPlaybackRate = 0.5
-          } else if (playbackRate >= 0) {
-            newPlaybackRate = 0.25
-          }
+    const standardShortcuts = disableShortcuts
+      ? []
+      : [
+          {
+            keyCode: 32, // spacebar
+            handle: () => togglePlay(),
+          },
+          {
+            keyCode: 75, // k
+            handle: () => togglePlay(),
+          },
+          {
+            keyCode: 70, // f
+            handle: () =>
+              videoService.send({type: 'TOGGLE_FULLSCREEN', element: rootElem}),
+          },
+          {
+            keyCode: 67, // c
+            handle: () => {
+              // focus the cue note input
+              cueFormElem?.input?.focus()
+              // start note taking session
+              videoService.send({type: 'TAKE_NOTE', source: 'shortcut'})
+              // pause video playback
+              videoService.send({type: 'CHANGE'})
+            },
```

Comment:
press `c` to focus on note input and start writing
`enter` to submit and continue playing the video